client/components/themes-list/index.jsx

```diff
@@ -148,70 +159,165 @@ function ThemeBlock( props ) {
 	);
 }
 
-function Empty( props ) {
-	const { wpOrgThemes, emptyContent, searchTerm, upsellCardDisplayed, translate } = props;
+function Options( { isFSEActive, recordTracksEvent, searchTerm, translate, upsellCardDisplayed } ) {
```

Comment:
This is a pretty big component; what do you think of moving it to its own file?

Reply:
We might even simplify the props by importing them into the end-consumer component rather than dragging them across multiple depth levels, but let's not overcomplicate this PR any further.